vocode/streaming/synthesizer/base_synthesizer.py

```diff
@@ -122,6 +123,7 @@ def __init__(self, synthesizer_config: SynthesizerConfigType):
                 synthesizer_config.sampling_rate == 8000
             ), "MuLaw encoding only supports 8kHz sampling rate"
         self.filler_audios: List[FillerAudio] = []
+        self.aiohttp_session = aiohttp.ClientSession()
```

Comment:
should be a parameter

Reply:
default arguments are evaluated when the function is defined, not when the constructor is evaluated - when importing BaseSynthesizer, I'd like not to not make a ClientSession